package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I believe this file should go in the `frontend` folder, right?

Reply:
It's right, I will delete it now 👌

Reply:
It's right, I'll remove it now 👌